contracts/subdaos/cwd-subdao-timelock-single/src/testing/tests.rs

```diff
@@ -222,7 +226,34 @@ fn test_execute_proposal() {
         res.messages
     );
     let updated_prop = PROPOSALS.load(deps.as_mut().storage, 10).unwrap();
-    assert_eq!(ProposalStatus::Executed, updated_prop.status)
+    assert_eq!(ProposalStatus::Executed, updated_prop.status);
+
+    // check proposal execution close_proposal_on_execution_failure = false
+    deps.querier.set_close_proposal_on_execution_failure(false);
+    let proposal = SingleChoiceProposal {
+        id: 10,
```

Comment:
let's have another proposal instance here for the following piece of test? i.e. it's already in `ProposalStatus::Executed` status by this time, and we test whether it's in the `ProposalStatus::Executed` status again below. that's an error-prone way of writing tests — such shadowing